llvm/test/CodeGen/X86/call-graph-section-assembly.ll

```diff
@@ -1,40 +1,43 @@
 ;; Test if temporary labels are generated for each indirect callsite with a callee_type metadata.
-;; Test if the .callgraph section contains the numerical callee type id for each of the temporary 
-;; labels generated. 
+;; Test if the .callgraph section contains the MD5 hash of callee type ids generated from
+;; generalized type id strings.
 
 ; RUN: llc -mtriple=x86_64-unknown-linux --call-graph-section -o - < %s | FileCheck %s
 
 ; CHECK: ball:
-; CHECK-NEXT: .Lfunc_begin0:
+; CHECK-NEXT: [[LABEL_FUNC:\.Lfunc_begin[0-9]+]]:
 define ptr @ball() {
 entry:
   %fp_foo_val = load ptr, ptr null, align 8
-   ; CHECK: .Ltmp0:
+   ; CHECK: [[LABEL_TMP0:\.Ltmp[0-9]+]]:
```

Comment:
Do you care that it's `.Ltmp`? I'd assume you just want to match anything after `.L`...

Reply:
I guess I don't. Update the test to match for `.L.*`. Thank you.